app.py

```diff
@@ -1,7 +1,9 @@
+# pylint: disable=no-member
+"""App routing and logic of application, application runs based off this file"""
 import os
-import flask
-from flask import Flask, render_template, redirect, flash, request, Blueprint
 from datetime import datetime
+import flask
+from flask import Flask, render_template, redirect, request
```

Comment:
we need flashes still!!

Reply:
I think since we're calling each flash as flask.flash we should only need to import flask in this case. Pylint treats it as an unused import. I can add it back, but it would get flagged by pylint.

Reply:
forgot about that, awesome!